presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -442,6 +448,11 @@ public static boolean isCollectColumnStatisticsOnWrite(ConnectorSession session)
         return session.getProperty(COLLECT_COLUMN_STATISTICS_ON_WRITE, Boolean.class);
     }
 
+    public static int getMaxPartitionsPerWriter(ConnectorSession session)
```

Comment:
usage of this method is missing. Have you forgot to push it?

Reply:
oh, yes. Get it added now. thank you @sopel39